packages/loader/container-loader/src/protocol/protocol.ts

```diff
@@ -125,7 +125,9 @@ export class ProtocolOpHandler implements IProtocolHandler {
 			}
 
 			case MessageType.Propose: {
-				// This should become unconditional once DeltaManager.processInboundMessage() stops parsing content (ADO #12052)
+				// DeltaManager is no longer parsing the contents, so we expect this to be a string here
+				// (However, switching this to an assert caused some tests to fail, apparently due to the same
+				// message object being processed multiple times - So we will retain the typeof check for now)
 				if (typeof message.contents === "string") {
```

Comment:
should this be an assert instead of check? I'd rather be more assertive.

Reply:
See the comment - tests started failing when I switched to an assert.  It was weird.

Reply:
Is there a follow up work item to investigate the issue found in tests?